README.md

```diff
@@ -36,98 +44,57 @@ bpftrace -e 'tracepoint:block:block_rq_issue { printf("%d %s %d\n", pid, comm, a
 # Count page faults by process
 bpftrace -e 'software:faults:1 { @[comm] = count(); }'
 
-# Count LLC cache misses by process name and PID (uses PMCs):
+# Count LLC cache misses by process and PID (uses PMCs):
```

Comment:
```suggestion
# Count LLC cache misses by thread name and PID (uses PMCs):
```
`comm` actually returns the name of the current thread. This is something that's been confusing users - didn't realise we had it wrong in the readme!

Reply:
TBH, I wasn't sure what to do about ALL of these examples in the readme as you'll notice that a lot of them say "by process" when grouping by `comm`. That's why I just removed the "name" in this heading.

Maybe it make sense to just change all the ones that use `comm` to say  "by thread".

Reply:
What's wrong with saying "name" as well? "By thread" could be interpreted to mean something more similar to "thread ID".

On a system with multiple Python processes, `comm` will return "python" for all of them, so aggregating on `comm` will group different threads and processes together because they share the same name.

Reply:
> What's wrong with saying "name" as well? 

I was just removing for consistency with the other examples above but let's not focus on that.

> "By thread" could be interpreted to mean something more similar to "thread ID".

Ok, so how about "by thread name" so it's consistent with how we talk about `comm`?

Reply:
Oh sorry, apparently I was too lazy to read your whole comment last time!